test/unit/core/simd/detail/builtin_simd_test.cpp

```diff
@@ -45,6 +48,41 @@ using uint32x8_t [[gnu::vector_size(32)]] = uint32_t;
 using uint64x4_t [[gnu::vector_size(32)]] = uint64_t;
 #endif
 
+namespace incomplete
```

Comment:
you and your namespaces 🤣  Is there any particular reason why this has to be in a namespace?
Why not call the struct `incomplete_template_type`?

Reply:
there was a difference between stuff within namespace and stuff without namspace. I test therefore both.

Reply:
Can you then please add a comment that this was intended.